src/services/commentService.js

```diff
@@ -0,0 +1,26 @@
+import { BASE_URL } from "../components/common/Constants";
+
+ const fetchCommentList = async (accessToken,taskId) => {
+    try {
+        
+      
+        const response = await fetch(BASE_URL + "/comment-info/comment-list/"+taskId, {
```

Comment:
Can you please keep the path also in Constant?
You can add the taskId in the commentService but the comment path should come from constant